modules/oe_whitelabel_paragraphs/oe_whitelabel_paragraphs.post_update.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for the OE Whitelabel paragraphs module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\oe_whitelabel\ConfigImporter;
+
+/**
+ * Add extra fields to description list paragraph.
+ */
+function oe_whitelabel_paragraphs_post_update_00001(array &$sandbox): void {
```

Comment:
there are no old versions of this module, so this will not run, what is in the install will run so that is enough. You can remove this file.

Reply:
Also removed the config/updates folder because that content is only used at the post_update.